quickstart/ia32-generic-qemu.md

```diff
@@ -82,14 +82,14 @@ qemu-system-i386 -hda _boot/phoenix-ia32-generic.disk -smp 4
 ```
 The number of detected cores is presented during kernel initialization.
 
-<img src="_images/qemu-ia32-generic-smp.png" width="600px">
+<img src="_images/qemu-ia32-generic-smp.png" width="700px">
 
 
 ## Running image on regular hardware
 To run the image on regular hardware please be sure that a target system is equipped with an ATA disk supporting the PATA interface. The image should be copied to the boot disk using the `dd` command (it is assumed that the target disk is represented by /dev/sda block device).
 
 ```
-  dd if=_boot/phoenix-ia32-generic.disk of=/dev/sda
+  dd if=_boot/ia32-generic-pc/phoenix.disk of=/dev/sda
```

Comment:
`ia32-generic-pc` is now a dedicated target for running our system on regular PC hardware. I'd create a separate quickstart page for it and move this section there.
The main difference between `qemu` and `pc` targets is that we may build and run virtio drivers on the former whereas they're not applicable to the latter.

Reply:
Yeah, I was thinking about it, but I don't have such hardware available now. I will opt for uploading this version and adding in near future separate quickstarts for `ia32-generic-pc` and `armv7a9-zynq7000-zturn` (when I could test them).